modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -606,6 +606,16 @@ TEST_P(Test_ONNX_layers, Pad2d_Unfused)
     testONNXModels("ZeroPad2d");
 }
 
+TEST_P(Test_ONNX_layers, LinearWithConstant)
+{
+    testONNXModels("lin_with_constant");
+}
+
+TEST_P(Test_ONNX_layers, MatmulWithTwoInputs)
+{
+    testONNXModels("matmul_with_two_inputs");
+}
```

Comment:
Please skip the tests for all the versions lower 2020.4: 
2020.3 - failed https://pullrequest.opencv.org/buildbot/builders/precommit_custom_mac/builds/1053
2020.4 - passed https://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/4279

Reply:
> Please skip the tests for all the versions lower 2020.4:
> 2020.3 - failed https://pullrequest.opencv.org/buildbot/builders/precommit_custom_mac/builds/1053
> 2020.4 - passed https://pullrequest.opencv.org/buildbot/builders/precommit_custom_linux/builds/4279

Thank you, I tried to reproduce it locally

Reply:
It's OK just add skip exception (see `applyTestTag`)